detection-rules/credential_phishing_docusign_embedded_image_lure.yml

```diff
@@ -34,6 +34,10 @@ source: |
       and not profile.by_sender().any_false_positives
     )
   )
+
+  //negate sender from Docusign
+  and not strings.like(sender.email.domain.root_domain, "docusign.com")
+
```

Comment:
```suggestion
```

I've added "docusign.com" to our $high_trust_sender_root_domains. Which should accomplish the intent yet still incorporates a dmarc failure check to prevent spoofed docusign senders.